Whiskey/Tasks/AppVeyorWaitForBuildJobs.ps1

```diff
@@ -0,0 +1,99 @@
+
+function Wait-WhiskeyAppVeyorBuildJob
+{
+    [CmdletBinding()]
+    [Whiskey.Task('AppVeyorWaitForBuildJobs')]
+    param(
+        [Parameter(Mandatory)]
+        [Whiskey.Context]$TaskContext,
+
+        [Parameter(Mandatory)]

```

Comment:
Setting a task property as mandatory via this attribute currently won't work without additional changes to `Invoke-WhiskeyTask` or `Get-TaskParameter` I think. If someone does not specify this attribute in their `whiskey.yml` then the build will hang as PowerShell prompts the user for this mandatory parameter value.